src/Widgets/DataFormNumberWidget/DataFormNumberWidgetComponent.tsx

```diff
@@ -9,17 +9,17 @@ import { OverlayTrigger, Popover } from 'react-bootstrap'
 import { DataFormNumberWidget } from './DataFormNumberWidgetClass'
 
 provideComponent(DataFormNumberWidget, ({ widget }) => {
-  const dataItem = useData().dataItem()
+  const dataItemAttribute = useData().dataItemAttribute()
```

Comment:
* [ ] FYI many `const dataItemAttribute` are only single-use temporary variables and could be inlined.